app/views/orders/edit.html.erb

```diff
@@ -0,0 +1,116 @@
+<ul class="row errors">
+  <% @cart.errors.each do |column, message| %>
+    <li>
+      <strong><%= column.capitalize %></strong> <%= message %>
+    </li>
+  <% end %>
+</ul>
+
+<% content_for :headline do %>
+         <h1> Order Form </h1>
+<% end %>
+<section class="double">
+     <%= form_for @cart do |f| %>
+          <section class="first-column">
+               <h2>Shipping Information</h2>
+               <%= f.hidden_field :order_state, value: "paid" %>
+               <%= f.label :name %><%= f.text_field :name %>
+               <%= f.label :email %><%= f.text_field :email%>
+               <%= f.label :street_address %><%= f.text_area :street_address%> <br>
+               <%= f.label :city %><%= f.text_field :city %>
+               <%= f.label :state %><%= f.select :state, [
```

Comment:
This drop down is cool! In the future - you could use "locales" and put this type of thing into a separate file so that it's not directly in your view